templates/patterns/page_header/page_header.ui_patterns.yml

```diff
@@ -47,3 +47,16 @@ page_header:
           label: "European Commission"
         - href: "#"
           label: "Announcements"
+    infos:
+      type: "array"
+      label: "Infos"
+      description: "A list of infos of the current page."
+      preview:
+        - text: "Monday 8 February"
+          icon_type: "general"
+          icon_name: "calendar"
+          icon_path: "/dist/images/icons/sprites/icons.svg"
+        - text: "Munich, Germany"
+          icon_type: "general"
+          icon_name: "location"
+          icon_path: "/dist/images/icons/sprites/icons.svg"
```

Comment:
We should hide all icon implementation details (such as type, name and path) in the pattern template since components using this pattern should not know how to build an icon.

I suggest using a generic `type` attribute instead and then, depending from the `type` we will map it to the correct icon, so something like:

```
    infos:
      type: "array"
      label: "Infos"
      description: "A list of infos of the current page."
      preview:
        - text: "Monday 8 February"
          type: "date"
        - text: "Munich, Germany"
          type: "location"
```

Reply:
Good catch. I will change too.